beater/burrowbeat.go

```diff
@@ -57,7 +57,28 @@ func (bt *Burrowbeat) Run(b *beat.Beat) error {
 		case <-ticker.C:
 		}
 
-		for _, group := range bt.groups {
+		logp.Debug("main", "Running tick")
+		groups := bt.groups
+		if len(groups) == 0 {
+			endpoint := "http://" + bt.host + ":" + bt.port + "/v2/kafka/" + bt.cluster + "/consumer"
+                        resp, err := http.Get(endpoint)
```

Comment:
You are mixing spaces and tabs in the indenting. Go standardizes on tabs, please convert this. You should also run `gofmt` to ensure that things are formatted correctly.